invoices/views.py

```diff
@@ -0,0 +1,46 @@
+from django.contrib.auth.mixins import LoginRequiredMixin
+from django.core.paginator import Paginator
+from django.shortcuts import get_object_or_404
+from rest_framework import viewsets
```

Comment:
`Explicit is better than Implicit`

from rest_framework.viewsets import ViewSet